.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,66 @@
+name: pre-release
```

Comment:
not sure we want this pre-release action on a public repository. everyone will be able to publish new packages, doesnt make much sense.
we need the pre-release appraoch as Manual action that will be executed on demand by developers

Reply:
Done: https://github.com/elementor/elementor-icons/pull/70/commits/0db1e5cd727ec5509a29e537b87a95fed6146f75